### DISCUSSION TITLE
Add comments in functions

### DISCUSSION THREAD

**Discussion by ssardina** [pacman-ai-projects/pacman-contest-cluster#107] [Ideas]
Add Python-standard comments to every function defined in the script...

**Comment by nirlipo** [comment]
Could use this plugin to help doing this: https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

![](https://github.com/NilsJPWerner/autoDocstring/raw/master/images/demo.gif)

**Comment by ssardina** [comment]
Yes! I use that, it is good and we should start doing it with at least the key methods
